editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java

```diff
@@ -91,13 +90,16 @@ public void initialize() {
         }
       });
 
-      // handle authentication for the service geodatabase
-      AuthenticationManager.setAuthenticationChallengeHandler(new DefaultAuthenticationChallengeHandler());
+      // set the user credentials required to authenticate with the service geodatabase
+      // alternatively authentication could be handled with an AuthenticationChallengeHandler
+      UserCredential userCredential = new UserCredential("editor01", "editor01.password");
 
       // create and load a service geodatabase
       serviceGeodatabase = new ServiceGeodatabase("https://sampleserver7.arcgisonline" +
         ".com/arcgis/rest/services/DamageAssessment/FeatureServer");
+      serviceGeodatabase.setCredential(userCredential);
       serviceGeodatabase.loadAsync();
+
```

Comment:
Only suggestion I can make here is perhaps creating the `UserCredential` immediately before setting it, so have it in order:
```
UserCredential userCredential = new UserCredential("editor01", "editor01.password");

// create a service geodatabase
 serviceGeodatabase = new ServiceGeodatabase("https://sampleserver7.arcgisonline" +
".com/arcgis/rest/services/DamageAssessment/FeatureServer");

// set the user credentials required to authenticate with the service geodatabase
UserCredential userCredential = new UserCredential("editor01", "editor01.password");
serviceGeodatabase.setCredential(userCredential);

// load the service geodatabase
serviceGeodatabase.loadAsync();
```

I don't know if it's strictly necessary to mention the `AuthenticationChallengeHandler`, but I haven't been following the discussion. I suppose the authentication itself is not an integral part of what the sample is 
trying to show? In which case maybe we can do without the comment

Reply:
@JonLavi Yeah exactly it was decided it wasn't necessary because it's not what the sample is demonstrating and was adding unnecessary steps for the user in showing off the branch versioning functionality. It's not a requirement to mention the authentication, so it's maybe overkill to add it into the comments, as there are clearly samples for authentication if someone wanted to add in the authentication. I'll reorder as you suggested and take that extra line out 👍